config/object_store/fields.yml

```diff
@@ -0,0 +1,6 @@
+email_address:
+  properties:
+    email_address:
+      type: string
```

Comment:
Is it going to be possible to validate that this input is, in fact, a valid email address etc.? Or do, say, relative validation against another field value?

Reply:
Yeah, it's definitely worth considering, but I wanted to get the pipework in first